### DISCUSSION TITLE
Website field uses different validation to Craft

### DISCUSSION THREAD

**Discussion by Coysh** [solspace/craft-freeform#1469] [Feature Requests]
Using a website field that creates an element in Craft.

I have a field in Freeform that is a website. The validation allows me to submit with a prefix (`https://`)
Craft doesn't allow URL fields without a prefix, so the error "Website is not a valid URL" is returned.

Freeform 5

**Comment by kjmartens** [comment]
Sorry for the delay @Coysh,

I have noted this for future consideration. We can look into building a transformer to handle this difference.

In the meantime, something to consider would be changing the field type from **Website** to **Regex** and configuring an expression that forced the HTTP/S prefix, etc. 🙂 

Something like this for the pattern as a starting point:

``` regex
/^(http|https):\/\//
```
